pheweb-test/README.md

```diff
@@ -0,0 +1,113 @@
+# Pheweb Integration
```

Comment:
If you plan to check this in (could also just leave this as a draft PR / branch), I would probably put this under api/genomics for now

Reply:
I would prefer to leave it as a draft/branch